Cloud Application Manager/Core Concepts/workspaces-and-collaboration.md

```diff
@@ -120,63 +124,113 @@ You can share a workspace of your Organization with other organization so that y
 
 This allows you to filter and search for the right scope to the work you need to do. The *Instances*, *Boxes* and *Providers* available to you will change to reflect the selected scope. For example, if you select a Cost Center, you can see all Instances, Boxes and Providers that are in all Workspaces belonging to that Cost Center.
 
 See more information about [**Context Switcher**](context-switcher.md).
 
 ### Sharing Boxes, Instances, and Providers
 
-When you create a box, launch an instance, or add a provider, they are created in the Workspace that you have selected in the Context Switcher. You can control how others use it by giving them view or edit access. Edit access gives users the same level of access as the owner but they cannot delete the asset.
+When you create a box, launch an instance, or add a provider, they are created in the Workspace that you are currently working in, that is, the one you have selected in the Context Switcher. You can control how others use it by giving them view or edit access. Edit access gives users the same level of access as the owner but they cannot delete the asset.
 
 * When you share with a user, the asset will appear in his personal workspace with the permission given.
 * When you share with a SAML or LDAP group, all users who belong to that group will be able to access the asset
-* If you decide to share with a team workspace, all the users on the workspace with the right permissions will be able to access the asset.
+* If you decide to share with a team workspace, all the users on the workspace with the corresponding permissions will be able to access the asset.
 
 When you share with a Workspace, all workspace members get access according to their role.
 
 A workspace is a shared environment in which members of that workspace can access the same providers, boxes, and instances. Workspaces are contextual. When you switch workspaces you see a whole different group of instances, boxes, and providers, which belong only to that workspace.
 
-#### Example
+In the sharing dialog of a resource, you can do the following:
+
+* Add a new user or workspace to the sharing list
+* Remove an existing user or workspace from the sharing list
+* Change access permissions from an existing user or workspace
+
+![Sharing dialog](../../images/cloud-application-manager/core-concepts/sharing-dialog.png)
+
+The changes to be applied to the sharing list appear highlighted with a dotted border line, and will be effective once you click on the **Save** button.
+
+### Sharing notifications
+
+The users that have been added to the sharing list of a resource, either a provider, box or instance, will see when accessing through the UI a notification badge consisting on a red dot that may include a number if several resources are available there. This badge will appear in several places to make him aware of the new resource he has access to:
```

Comment:
```suggestion
A notification badge consisting on a red dot will appear on the UI for those users users that have been added to the sharing list of a resource (i.e. provider, box or instance). The badge may include a number if two or more resources are available there. It will appear in several places to make the user aware of the new resource he has access to:
```